index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="style.css">
+	<script src="https://kit.fontawesome.com/yourcode.js" crossorigin="anonymous"></script>
+	<title>Portfolio</title>
+</head>
+<body>
+	<section class="screen_container">
+		<header class="toolbar_container">
+			<a href="#" class="Title_logo">My Logo</a>
+			<nav>
+				<img src="./images/i.png"  class="logo" alt="Icon menu">
+			</nav>
+		</header>
+
+		<main class="main_container">
+			<section class="text_container">
+				<h1 class="paragraphe">Hey There. <br/>I’m Leader.</h1>
+				
+				<h4>I'm a Software Developer</h4>
+				<p class="para">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+					don’t hesitate to contact me.</p>
+			</section>
+			
+			<section class="icons-container">
```

Comment:
- [ ] This section is not needed. for improved semantics, kindly consider removing this second section. everything in the here would be inside this `<section class="text_container">`.
- [ ] After including the `ul` in the above mentioned section, you can place this ul as it in the figma design. just use margins and paddings, and kindly consider removing the `position: absolute; and right and left because its not needed`. you can achieve the design just by using margins and paddings to the `<ul class="icons-containers"> `

Reply:
Fixed.

Reply:
please i will try but position help me to this part but in the next one I won't use it anymore